src/StockportWebapp/Views/stockportgov/Home/Index2023.cshtml

```diff
@@ -6,6 +6,7 @@
     ViewData["og:title"] = "Welcome to Stockport Council";
     ViewData["Description"] = @Model.HomepageContent.MetaDescription;
     Layout = "../Shared/_LayoutHome2023.cshtml";
+    var SpotlightOnClasses = Model.HomepageContent.CallToAction is not null ? "section section--no-margin section--padded-bottom section--grey" : "section section--white";
```

Comment:
Could you maybe have this on 3 lines, put the ? "sec... etc on the 2nd line and then the : "sec... etc on the 3rd, just for better readability?